src/pages/index.js

```diff
@@ -61,9 +132,22 @@ function handleProfileEditOpen() {
 }
 
 function handleProfileEditSubmit(data) {
-  userInfo.setUserInfo({ name: data.name, job: data.description });
-  profileFormValidator.resetValidation();
-  profileFormPopup.close();
+  profileFormPopup.submitText("Saving...");
+  api
+    .editUserInfo(data)
+    .then((data) => {
```

Comment:
` .then((data) => {`


Please, name the response from the server here `userData` not to confuse it with the data you get in line 134 (the inputs values)


```js

    .then((userData) => {
      userInfo.setUserInfo({
        name: userData.name,
        job: userData.about,
        avatar: userData.avatar,
      });
```